src/style.css

```diff
@@ -0,0 +1,66 @@
+body {
+  margin: 0;
+
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 100px;
+  color: rgb(255, 255, 255);
+}
+
+.container {
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.item {
+  display: flex;
+  flex-grow: 1;
+  justify-content: center;
+  align-items: center;
+  width: 300px;
+  height: 300px;
+}
+
+.item--1 {
+  background-color: rgb(255, 0, 0);
+}
+
+.item--2 {
+  background-color: rgb(204, 0, 0);
+}
+
+.item--3 {
+  background-color: rgb(153, 0, 0);
+}
+
+.item--4 {
+  background-color: rgb(102, 0, 0);
+}
+
+.item--5 {
+  background-color: rgb(51, 0, 0);
+}
+
+.item--6 {
+  background-color: rgb(0, 0, 0);
+}
+
+@media (min-width:600px) and (max-width:899px) {
+  .item--3,
+  .item--5,
+  .item--6 {
+    order: 2;
+  }
+}
+
+@media (min-width:900px) {
+  .item {
+    width: 33.3%;
```

Comment:
imo, it'd be better to use flex-basis, if you need to swap your flex-direction at some point)